zanata-client-commands/src/main/resources/prompts.properties

```diff
@@ -91,3 +91,21 @@ unrecognized.variables=Valid variables are %s. Anything else in '%s' will be tre
 confirm.rule=Continue with current mapping rule(s) (y/n)?
 
 locales.in.config.deprecated=Locale mappings are now handled using locale aliases on the server, so locale mappings in the project config file (zanata.xml) are now deprecated.\nPlease add a locale alias in the project language settings to replace each locale mapping in zanata.xml, then remove the <locales> section from zanata.xml.
+
+# {0} - fileType, {1} - file extension, {2} - fileType
+file.type.suggestFromExtension=Please use file type '{0}' for extension '{1}', e.g --file-types {2}
+# {0} - fileType
+file.type.duplicateFileType=File type '{0}' cannot be used more than once in option --file-types.
+# {0} - file extension, {1} - file type
+file.type.notSupportedExtension=Extension '{0}' for file type '{1}' is not supported by the target server and will be ignored.
+# {0} - fileType
+file.type.typeNotSupported=Requested file type '{0}' is not supported by the target server and will be ignored.
+# {0} - fileType1, {1} - fileType2, {2} - extension
+file.type.conflictExtension=Conflict file type between '{0}' and '{1}' for extension '{2}' in option --file-types.
```

Comment:
- [ ] I think we can state this more plainly, like "You used extension '{2}' in both '{0}' and '{1}' in option --file-types. Extensions can only be used for one type."